implementation/scripts/.util/tools.sh

```diff
@@ -129,7 +129,7 @@ function util::tools::packager::install () {
 
     if [[ ! -f "${dir}/packager" ]]; then
       util::print::title "Installing packager"
-      GOBIN="${dir}" go get -u github.com/cloudfoundry/libcfbuildpack/packager
+      GOBIN="${dir}" go install github.com/cloudfoundry/libcfbuildpack/packager
```

Comment:
Reading the release notes for 1.18, do we need also include the `@latest` version suffix?

```suggestion
      GOBIN="${dir}" go install github.com/cloudfoundry/libcfbuildpack/packager@latest
```